### PR TITLE
Zoe Release Candidate Updates

### PR DESCRIPTION
**Pull request by katelynsills**
Current status: tests pass and the deploy scripts do not error, but the application does not yet work.

#agoric-sdk-branch: zoe-release-candidate

### CODE REVIEW

**Comment by katelynsills** [comment]
I can't review my own PR, but looks good to me!

File: api/deploy.js [hunk 1]
```diff
@@ -119,21 +119,20 @@ export default async function deployApi(homePromise, { bundleSource, pathResolve
   console.log('Installing contract');
   const issuerKeywordRecord = harden({ Tip: tipIssuer });
   const {
-    creatorInvitation: adminInvite,
-    creatorFacet
+    creatorInvitation: adminInvitation,
+    instance,
   } = await E(zoe).makeInstance(encouragementInstallation, issuerKeywordRecord);
-  const instance = await E(creatorFacet).getInstance();
   const publicAPI = E(zoe).getPublicFacet(instance);
```
Comment (dtribble) [inline api/deploy.js:128]: This can now be in the `makeInstance` result above?

**Comment by dtribble** [comment]
LGTM

File: api/deploy.js (hunk 1 above)
Comment (michaelfig) [reply api/deploy.js:128]: Good catch.  Will move.

**Pull request merged**